test/appium/views/profile_view.py

```diff
@@ -563,6 +563,16 @@ def add_custom_network(self):
         self.confirm_button.click()
         return self.get_sign_in_view()
 
+    def change_network(self, network_name):
```

Comment:
There is previously added method **switch_network** doing the same: https://github.com/status-im/status-react/blob/develop/test/appium/views/profile_view.py#L533